reconcile/aus/base.py

```diff
@@ -142,13 +143,25 @@ def process_org(
             )
 
     def get_upgrade_specs(self) -> dict[str, dict[str, OrganizationUpgradeSpec]]:
-        return {
-            ocm_env.name: self.get_ocm_env_upgrade_specs(
-                ocm_env,
-                self.params.ocm_organization_ids,
-            )
-            for ocm_env in self.get_ocm_environments()
-        }
+        envs_org_upgrade_specs: dict[str, dict[str, OrganizationUpgradeSpec]] = {}
+        for ocm_env in self.get_ocm_environments():
+            try:
+                envs_org_upgrade_specs[ocm_env.name] = self.get_ocm_env_upgrade_specs(
+                    ocm_env,
+                    self.params.ocm_organization_ids,
+                )
+            except Exception as e:
+                logging.exception(
+                    f"Failed to get org upgrade specs for OCM environment {ocm_env.name}. Skipping.",
+                    e,
+                )
```

Comment:
This logging may have another exception, either all f-string, or use preferred logging format string, [doc](https://docs.python.org/3/howto/logging.html#optimization):

```python
logging.exception("Failed to get org upgrade specs for OCM environment %s. Skipping. %s", ocm_env.name, e)
```

Reply:
changed. ty!